lib/ovirt_metrics/models/vm_device_history.rb

```diff
@@ -0,0 +1,10 @@
+module OvirtMetrics
+  class VmDeviceHistory < OvirtHistory
+    # have to rename the inheritance_column since this table has a column
+    # called "type"
+    self.inheritance_column = "other_type"
```

Comment:
So we did this because we need to use vm_device_history and it defines a "type" column which messes things up. No other table we saw in the schema has a column with this name. Any way to avoid confusion we will write this as :_type_disabled